slither/detectors/naming_convention/naming_convention.py

```diff
@@ -81,7 +81,11 @@ def detect(self):
 
                 for argument in func.parameters:
 
-                    if self.is_mixed_case(argument.name) is False:
+                    argument_name = argument.name
+                    if argument_name and argument_name.startswith('_'):
+                        argument_name = argument_name[1::]
```

Comment:
```suggestion
                        argument_name = argument_name[1:]
```

or eventually: `argument_name = argument_name.lstrip('_')` but that would remove all underscores from the begining.